src/SaleorAuthClient.ts

```diff
@@ -179,6 +184,7 @@ export class SaleorAuthClient {
   signOut = () => {
     this.accessToken = null;
     this.storageHandler?.clearAuthStorage();
+    document.cookie = "token=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;";
```

Comment:
You can use `cookie.serialize` instead:
```suggestion
    document.cookie = cookie.serialize("token", "", { expires: new Date(0) });
```